### DISCUSSION TITLE
How to fix plotting nodal displacement (with show_displacement)

### DISCUSSION THREAD

**Discussion by Seb-Pi** [pyansys/pymapdl-reader#316] [Q&A] [answered]
Hello!

I have some issue with viewing my results and not found this problem here anywhere, so i felt free to open a new discussion.
I've made a simple rack with BEAM188 Elements and want view it's static deformed shape.
So far so good, but if i add extra a third node to BEAM188 (to define rotation), the reader can somehow not handle it?
it seems it is trying to plot displacements for the thrid nodes, but there is no calculated displacement for this nodes...

Viewing results without "show_displacement=True" works.
In MAPDL it shows correctly, so there seems to be no issue with the file.

So this is the very simple code: 
```
from ansys.mapdl.reader.rst import Result
import numpy as np
rst_file = 'file.rst'

rst = Result(rst_file)
max_disp = np.abs(rst.nodal_displacement(0)[1][:, :3]).max()
rst.plot_nodal_displacement(0)
rst.plot_nodal_displacement(0, displacement_factor=1/ max_disp, show_displacement=True, overlay_wireframe=True)
```
You can find the resut file here:
[result.zip](https://github.com/ansys/pymapdl-reader/files/12538227/result.zip)

I don't know if i do something wrong here, so i am looking forward for your help!

**Comment by mikerife** [comment, marked as answer]
Hi @Seb-Pi 
You have it correct - the PyMAPDL Reader does not handle beam orientation nodes very well (yet).  See here for an example and an sort-of solution:
https://github.com/ansys/pymapdl-reader/issues/232
